frontend/components/atoms/gutenberg/_colors.scss

```diff
@@ -1,12 +1,8 @@
 $guten-colors: (
   'white': $white,
-  'gray-1': #F5F5F5,
```

Comment:
We usually need gray scale colors, and even in the Blocks color palette they'll be more useful for the admin, why don't we preserve them here?

Reply:
why would you need the gray colors? they are not colors from the theme styleguide

Reply:
For the editor they'll be useful for adding text blocks with gray background. Almost in every project I've worked on, there are text blocks with gray background, or even gray text. Obviously they can be edited to meet the styleguide, or even deleted. But could work as starting point. Anyway if you think they are not useful, let's remove them.

Reply:
with white, black, and the primary and secondary colors from the theme is fine I think